pkgs/by-name/el/element-call/package.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, mkYarnPackage
+, fetchFromGitHub
+, fetchYarnDeps
+}:
+
+let
+  os = if stdenv.isDarwin then "osx" else "linux";
+  offlineCacheHash = {
+    linux = "sha256-xVpObAJEsHVS0+bvasQZ97PRRUdtvXpiV03qt29tz8c=";
+    osx = "sha256-bHc1FjvKDTL5apJqgXv7zk1z1WF68B3T7ibV7ZX3DWM=";
+  }.${os};
+in
+mkYarnPackage rec {
+  pname = "element-call";
+  version = "0.5.15";
+
+  src = fetchFromGitHub {
+    owner = "element-hq";
+    repo = "element-call";
+    rev = "v${version}";
+    hash = "sha256-2RGzbbdlDj497YFRdppegW1P9O2H4mQ5vfhZEcfMRkc=";
+  };
+
+  packageJSON = ./package.json;
+
+  prePatch = ''
+    patch -i ${./name.patch}
+  '';
+
+  offlineCache = fetchYarnDeps {
+    yarnLock = "${src}/yarn.lock";
```

Comment:
That could be triggering IFD, you are sure it does not?

Reply:
I'm not sure, though this is what the documentation recommends doing https://github.com/NixOS/nixpkgs/blob/1461dbbee35356d6aea102da072df380b0a5749c/doc/languages-frameworks/javascript.section.md?plain=1#L324